tempodb/blocklist/poller.go

```diff
@@ -95,6 +98,15 @@ func (ownsNothingSharder) Owns(_ string) bool {
 	return false
 }
 
+// OwnsEverythingSharder owns everything. You do not want this developer on your team.
+var OwnsEverythingSharder = ownsEverythingSharder{}
```

Comment:
unused? remove?

Reply:
This is useful for building binaries outside of tempo main.  I'd like to keep it for this purpose.

Reply:
I don't think I understand? If fa dependent codebase wants an `OwnsEverythingSharder` it can make it itself. We shouldn't keep this code unless we're using it.

Reply:
I may use this in the test that I mentioned I'd write.  Let's see.

Reply:
Can you move it to the test then?